src/modules/main/Main.tsx

```diff
@@ -33,6 +36,9 @@ const Main = ({ match }: RouteChildrenProps) => {
 
           <Route exact path={`${match.path}/projects`} render={() => <div>projects</div>} />
 
+          <Route path={`${match.path}/admin/:category`} component={AdminPanel} />
+          <Redirect from={`${match.path}/admin`} to={`${match.path}/admin/${AdminTabCategory.TECHNOLOGIES}`} />
```

Comment:
You should add redirection like that inside AdminPanel with hook - check for ref - useRouteValidation. Why this is better ? Because this feature is related to this module and also easier to test this inside this module